docs/practices/docs.md

```diff
@@ -0,0 +1,100 @@
+# Documentation
+
+This chapter describes nearcore approach to develope documentation. In general,
+there are three primary types of documentation to keep in mind:
+
+* **Protocol specification** ([source](https://github.com/near/NEPs),
+  [rendered](https://nomicon.io)) is a formal description of the NEAR protocol.
+  It can be used to implement alternative NEAR clients.
+* **User Docs** ([rendered](https://docs.near.org)) explain how to use the near
+  network. User docs are also split into documentation for validators (that is,
+  how to run your own near node) and documentation for smart contract
+  developers.
+* **Internal Development Docs** ([rendered](https://near.github.io/nearcore/),
+  [source](https://github.com/near/nearcore/tree/master/docs)) is the book you
+  are reading right now! The target audience here are developers of nearcore.
+
+Here, we focus on the last sort -- internal docs.
```

Comment:
I think we can remove this line.  We say above that this book is about internal docs.

```suggestion
```